anvio/__init__.py

```diff
@@ -2019,6 +2019,15 @@ def get_args(parser):
                      "class based on the number of threads. If you have *any* hesitation about whether you know what you are doing, "
                      "you should not change this value."}
                 ),
+    'window-range': (
+            ['--window-range'],
+            {'default': 500,
+             'metavar': 'WINDOW_RANGE',
```

Comment:
`metavar` could be `INT`

Reply:
Sorry Meren I tried replacing `str` with `INT` but got an error. The user is supposed to enter in x:y (x = smallest window size, y = largest window size) which gets parsed in `sanity check` as a string. So I think using `INT` won't work, but honestly I don't think I understand `metavar` enough.

Reply:
metavar is nothing but giving the user an idea about what is expected as input for that variable. You can write anything there and it will show up in the help menu output.